README.md

```diff
@@ -1 +1,22 @@
-# termionus
\ No newline at end of file
+# termionus
+
+We are will eventually create a logo but for now you can look at this >\_
+
+## What is termionus?
+
+termionus is an open source terminal emulator for all operating systems.
+The main focus of termionus is to provide great user experience for people who
+are tired of slaw and underdeveloped terminals that are maid to work and not to be enjoyed.
```

Comment:
I wouldn't want to describe `termionus` as the "ultimate solution" to the available terminal emulators on the market, as it sounds too rude and unprofessional. Let's go with a more neutral response:

```suggestion
The main focus of `termionus` is to provide great user experience for developers, as well as provide a good support for several types of programs and available shells.
```